### ISSUE TITLE
#a11y - Add an existing account and Sign out should be <button>

### ISSUE THREAD

**Issue by luce-carevic** [elk-zone/elk#1366]
![2023-01-21-Language  Settings  Elk](https://user-images.githubusercontent.com/6073336/213863961-9a74634c-aa0c-4850-b900-6a080f0b5906.png)

![2023-01-21-Outils de développement - Language  Settings  Elk - httpselk zonesettingslanguage](https://user-images.githubusercontent.com/6073336/213863791-ee333f42-34b5-488b-8fd0-1fd67c5ca670.png)

These interactive elements are not keyboard accessible and they should be `<button>`.

See [Just use button](https://www.youtube.com/watch?v=CZGqnp06DnI).
